### ISSUE TITLE
Update channel, after saving changes continues to revert back to CentOS.

### ISSUE THREAD

**Issue by vulcanjedi** [Tautulli/Tautulli#797]
AMAZING work you guys.
Just curious if PlexPy has this information, can we get a lazy button to post or auto post to GH?

**Version:**
VERSION 1.4.8
**Branch:**
master
**Commit hash:**
    2150961d2742afc24d5982fe3d0ddee141f89a26
**Operating system:**
ubuntu  Linux 3.13.0-91-generic
**Python version:**
    2.7.6 (default, Jun 22 2015, 17:58:13) [GCC 4.8.2]
**What you did?**
Trying to update the plex Updates settings

**What happened?**
Navigated to \plex media server.
Enabled Monitor Plex updates.
Update channel Plex Pass
Release Linux Ubuntu 64 bit. I have done this repeatedly as I have never recalled seening an update notification, but as I continue to go into this screen, after hitting save...everything seems fine but upon leaving and returning and releasing cache and reloading page... the release always reverts back to CentOS?

**What you expected?**
Setting to be sticky on what I set, which is Linux Ubuntu 64bit

**How can we reproduce your issue?**

launch plexpy admin gui web page
navigate to Plex Media server.
plex pass update channel
Linux ubuntu 64 bit release
SAVE
navigate somewhere else on the left tree. / close browser / purge cache
validate the setting has not been reverted to CentOS

**What are your (relevant) settings?**

**Link to logs:**

**Comment by JonnyWong16** [comment]
After saving the settings, what is the value of `pms_update_distro_build` in your `config.ini` file?

**Comment by vulcanjedi** [comment]
I just repeated, the GUI still says CentOS (after saving and revisiting page), but the config.ini does in fact say ubuntu. 
Interesting. I also checked my notification history and there was no notification, the last update for PMS came out July21 and I updated very close to it releasing.

pms_update_channel = plexpass
pms_plexpass = 1
pms_version = 1.0.2.2413-7caf41d
pms_update_distro_build = linux-ubuntu-x86_64
pms_platform = Linux
